webapp/src/views/projects/import/ImportView.tsx

```diff
@@ -83,6 +88,7 @@ export const ImportView: FunctionComponent = () => {
         ],
       ]}
       maxWidth="wide"
+      overflow="scroll"
```

Comment:
I think this makes the scrollbar always visible on windows even if not necessary, is it intended? I usually use "auto".